src/sentry/incidents/logic.py

```diff
@@ -629,6 +631,13 @@ def create_alert_rule(
             detection_type=detection_type,
         )
 
+        if alert_rule.detection_type == AlertRuleDetectionType.DYNAMIC.value:
```

Comment:
Can we move this check below the project checks?

I don't think multiple projects is utilized atm, but since the logic exists maybe we implement this assuming that multiple projects per rule is valid today?

Reply:
The `include_all_projects` part or the `AlertRuleProjects` creation or the `alert_rule.subscribe_projects` part?

Reply:
I moved it to below `include_all_projects` but lmk if you wanted it lower